### PR TITLE
DOCKER: added docker

### PR DESCRIPTION
**Pull request by kenu21**

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+    ports:
+      - $SPRING_LOCAL_PORT:$SPRING_DOCKER_PORT
+      - $DEBUG_PORT:$DEBUG_PORT
+    environment:
+      SPRING_APPLICATION_JSON: '{
+        "spring.datasource.url"  : "jdbc:mysql://mysqldb:$MYSQL_DOCKER_PORT/$MYSQLDB_DATABASE",
+        "spring.datasource.username" : "$MYSQLDB_ROOT_USER",
+        "spring.datasource.password" : "$MYSQLDB_ROOT_PASSWORD"
+      }'
+      JAVA_TOOL_OPTIONS: "-agentlib:jdwp=transport=dt_socket,server=y,suspend=n,address=*:5005"
```
Comment (liliia-ponomarenko) [inline docker-compose.yml]:
```suggestion
      JAVA_TOOL_OPTIONS: "-agentlib:jdwp=transport=dt_socket,server=y,suspend=n,address=*:5005"
```
better use $DEBUG_PORT variable

**Review by liliia-ponomarenko — APPROVED** [review]
Good job! Let’s  add a  .env.template file with the list of variables (without value)

**Comment by kenu21** [comment]
> Good job! Let’s add a .env.template file with the list of variables (without value)

@liliia-ponomarenko 
I don't think I should push this file to GitHub. I have this file local:
https://github.com/kenu21/book-shop/pull/12/files#diff-bc37d034bad564583790a46f19d807abfe519c5671395fd494d8cce506c42947R6

File: docker-compose.yml (hunk 1 above)
Comment (kenu21) [reply docker-compose.yml]: Changed

**Pull request merged**